content/03-reference/01-tools-and-interfaces/04-introspection.mdx

```diff
@@ -35,15 +35,41 @@ You can learn more about how Prisma maps types from the database to the types av
 
 You can introspect your database using the `prisma introspect` command of the [Prisma CLI](prisma-cli/installation). Note that using this command requires your [connection URL](../database-connectors/connection-urls) to be set in your Prisma schema!
 
-> **Warning**: The `prisma introspect` command overwrites the current version of your Prisma schema! If you made any manual adjustments to the Prisma schema (e.g. by adding [comments](prisma-schema/prisma-schema-file), [making a 1-1-relation required on both sides](prisma-schema/relations#making-both-sides-required-in-one-to-one-relations) or [configuring your Prisma Client API](prisma-client/configuring-the-prisma-client-api)), be sure to back up your schema before running the command!
-
 Here's a high-level overview of the steps that `prisma introspect` performs internally:
 
 1. Read the [connection URL](../database-connectors/connection-urls) from the `datasource` configuration in the Prisma schema
 1. Open database connection
 1. Introspect database schema (i.e. read tables, columns and other structures ...)
 1. Transform database schema into Prisma data model
-1. Write data model into Prisma schema
+1. Write data model into Prisma schema or [update existing schema](#re-introspection)
+
+## Re-introspection
+
+You **re-introspect** when you run the `prisma introspect` command with an existing Prisma schema. In version 2.6.0 and later, re-introspection merges manual changes made to the schema with changes made in the database. Re-introspection maintains the following manual changes:
+
+- Order of `model` blocks
+- Order of `enum` blocks
+- Comments
+- `@map` and `@@map` attributes
+- `@updatedAt`
+- `@default(cuid())` (`cuid()` is a Prisma-level function)
+- `@default(uuid())` (`uuid()` is a Prisma-level function)
+- Custom `@relation` names
+
+The following properties of the schema are determined by the database:e
+
+- Order of columns within `model` blocks
+- Order of values within `enum` blocks
+
+> **Note**: All `enum` blocks are listed below `model` blocks.
+
+### Force overwrite
```

Comment:
Maybe this could refer to another use case for this: When you have an invalid schema, you _need_ to use `--force` to allow Introspection to work.